src/usb_moded-util.c

```diff
@@ -300,72 +300,64 @@ static int util_get_hiddenlist (void)
 
 static int util_handle_network(char *network)
 {
-    char *operation = 0, *setting = 0, *value = 0;
-    DBusMessage *req = NULL, *reply = NULL;
-    char *ret = 0;
+    int result = EXIT_FAILURE;
+    DBusMessage *req = NULL;
+    DBusMessage *reply = NULL;
+    const char  *ret = NULL;
 
-    operation = strtok(network, ":");
+    const char *operation = strtok(network, ":");
+    const char *setting = strtok(NULL, ",");
     printf("Operation = %s\n", operation);
-    setting =  strtok(NULL, ",");
     printf("Setting = %s\n", setting);
-    value =  strtok(NULL, ",");
-    printf("Value = %s\n", value);
-    if(operation == NULL || setting == NULL )
+
+    if(!operation || !setting)
```

Comment:
Everywhere else there is `if( !operation || !setting )` style. I guess style change here was unintentional?

Reply:
Think: patch to a random problem function spotted while going through control flows somewhat related to the actual thing to fix -> the idea is/was to keep lines that do not need to be modified as they were to retain some context in diff -> follow whatever whitespace rules are used within that one function, which happens to be inconsistent in whitespace usage to start with

Reply:
These were just minor nitpicks anyways, I just thought that something was forgotten. I'm ok with this if you are.